src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -138,10 +141,16 @@ export default class DiscussionArea {
    * Posts the comment in the new comment area, and reloads the discussion.
    */
   postNewComment() {
+    /* eslint-disable indent */
+    const type = DiscussionArea.#ELEMENT_NEW_COMMENT_TYPES
```

Comment:
type is generic please make it more specific

Reply:
renamed to `commentType`